enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -617,6 +622,35 @@ def get_assignments_serializer(self, obj):
             context=context,
         )
         return serializer.data
+    
+    @extend_schema_field(LearnerCreditRequestSerializer(many=True))
+    def get_learner_requests(self, obj):
+        """
+        Return serialized learner credit requests associated with the user and policy's enterprise customer.
+        
+        Returns:
+            list: Serialized learner credit requests if policy has BNR enabled and user is authenticated.
+                  Empty list otherwise.
+        """
+        # Early returns if BNR not enabled or no user ID
+        if not obj.bnr_enabled:
+            return []
+            
+        lms_user_id = self.context.get('lms_user_id')
+        if not lms_user_id:
+            return []
+
+        # Get active learner credit requests for this user and enterprise customer        
+        learner_requests = LearnerCreditRequest.objects.filter(
```

Comment:
Noticed that we have missed other states like `Pending`. Is it intentional?

Reply:
+1

Reply:
We are supposed to consider only open requests that will be `requested` and `approved`.